wpk/generate_wpk_package.sh

```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+INSTALLATION_PATH="/var"
 
 trap ctrl_c INT
 
 function build_wpk_windows() {
   local BRANCH="$1"
```

Comment:
Please, try to the same format that below "${1}

Reply:
All unified to ${} https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9